docs/getting_started/02_breakdown.md

```diff
@@ -35,25 +35,36 @@ _Verifier.toml_ contains public in/output values computed when executing the Noi
 
 _Nargo.toml_ contains the environmental options of your project. It contains a "package" section and a "dependencies" section.
 
-#### Package section
-
-The package section requires a number of fields including:
-
-- name - the name of the package
-- type - can be "bin", "lib", or "contract" to specify whether its a binary, library or Aztec contract
-- authors
-- compiler_version - specifies the version of the compiler to use. This is not currently enforced by the compiler, but will be in future versions.
-
-For example:
+Example Nargo.toml:
 
-```toml
+```yaml
 [package]
 name = "noirstarter"
 type = "bin"
 authors = ["Alice"]
 compiler_version = "0.9.0"
+
+[dependencies]
+ecrecover = {tag = "v0.9.0", git = "https://github.com/colinnielsen/ecrecover-noir.git"}
+```
+
+Nargo.toml for a [workspace](../modules_packages_crates/workspaces) will look a bit different. For example:
+
+```yaml
+[workspace]
+members = ["crates/a", "crates/b"]
+default-member = "crates/a"
 ```
 
+#### Package section
+
+The package section requires a number of fields including:
+
+- name - the name of the package
+- type - can be "bin", "lib", or "contract" to specify whether its a binary, library or Aztec contract
+- authors - authors of the project
+- compiler_version - specifies the version of the compiler to use. This is not currently enforced by the compiler, but will be in future versions.
```

Comment:
These are both optional now

The full list, with required/optional:
- `name` (required)
- `type` (required)
- `entry` (optional) - a relative filepath to use as the entry point into your package (overrides the default of `src/lib.nr` or `src/main.nr`)
- `description` (optional)
- `authors` (optional)
- `compiler_version` (optional) - this might be made required in the future when we support comparing the compiler versions
- `backend` (optional)
- `license` (optional)
-